src/main/java/com/google/devtools/build/lib/analysis/config/CoreOptions.java

```diff
@@ -342,14 +342,30 @@ public String getTypeDescription() {
       documentationCategory = OptionDocumentationCategory.OUTPUT_PARAMETERS,
       effectTags = {OptionEffectTag.ACTION_COMMAND_LINES},
       help =
-          "Specifies the set of environment variables available to actions. "
+          "Specifies the set of environment variables available to actions performed using target configuration. "
```

Comment:
Maybe "with target configuration" sounds a bit simper? Same below.

Reply:
To be "available to actions with target configuration." ?

Reply:
Yes, that's what I mean